### PR TITLE
So0k support launchtemplates

### PR DESCRIPTION
**Pull request by holtwilkins**

### CODE REVIEW

**Comment by holtwilkins** [comment]
Fixes https://github.com/palantir/bouncer/issues/31

**Comment by so0k** [comment]
oops, sorry for not handling some edge cases and missing out a few pointer dereferences - I thought about doing unit tests but ...

File: bouncer/asg.go [hunk 1]
```diff
@@ -40,7 +40,7 @@ func NewASG(ac *aws.Clients, desASG *DesiredASG, force bool, startTime time.Time
 	var instances []*Instance
 
 	for _, asgInst := range awsAsg.Instances {
-		inst, err := NewInstance(ac, awsAsg, asgInst, awsAsg.LaunchConfigurationName, force, startTime, desASG.PreTerminateCmd)
+		inst, err := NewInstance(ac, awsAsg, asgInst, force, startTime, desASG.PreTerminateCmd)
```
Comment (holtwilkins) [inline bouncer/asg.go]: Fixed this up @so0k .
